tests/components/figure.test.ts

```diff
@@ -64,8 +68,20 @@ class TextFigure extends Component<Props, SpreadsheetChildEnv> {
   static template = TEMPLATE;
 }
 
+mockChart();
+mockGetBoundingClientRect({
+  "o-popover": () => ({ height: 0, width: 0 }),
+  "o-spreadsheet": () => ({ top: 100, left: 200, height: 1000, width: 1000 }),
+  "o-figure-menu-item": () => ({ top: 500, left: 500 }),
+});
+
 beforeAll(() => {
   figureRegistry.add("text", { Component: TextFigure });
+  chartComponentRegistry.add("line", ChartJsComponent);
+  chartComponentRegistry.add("bar", ChartJsComponent);
+  chartComponentRegistry.add("pie", ChartJsComponent);
+  chartComponentRegistry.add("gauge", ChartJsComponent);
+  chartComponentRegistry.add("scorecard", ScorecardChart);
```

Comment:
This is foireux, this wasn't needed before because we imported `charts.test` where, probably by luck, the registries were populated.
And I cannot import `charts.test` anymore, sicne I need to mockGetBoundingClientRect in both files.

We should do something about that, there is also a random `chartComponentRegistry.add("bar", ChartJsComponent);` in `grid.test.ts`